### DISCUSSION TITLE
Setting master branch to Rolling

### DISCUSSION THREAD

**Discussion by fmrico** [PlanSys2/ros2_planning_system#267] [General]
Dear PlanSys2 users and devs,

After releasing PlanSys2 for Iron, I am considering setting the `master` branch to `Rolling`, which is usual in other reference packages, like Nav2, and convenient to incorporate the last improvements in ROS 2.

I would like your opinions before doing it at this week's final.

Best
Francisco

**Comment by robodrome** [comment]
I think it's good to align with reference packages like Nav2 and the likes.
However, the [Nav2 github repo](https://github.com/ros-planning/navigation2/branches) does not seem to have a `rolling` branch. It is `main`. The [ROS2 repo](https://github.com/ros2/ros2/branches) however does have a `rolling` branch.

So what to choose ... there is an old discussion about it [here](https://discourse.ros.org/t/change-default-branch-name-to-rolling-in-ros-2-core/26009).

Best,
David

**Reply by fmrico** [reply]
Yes, the idea is that main is built with rolling

**Comment by fmrico** [comment]
Hi again!! After thinking about it, I decided to move `main` to `rolling` as the development and default branch. That would clearly state with which distro we should build plansys2, and at the same time we are ready to create packages for incoming distros. I will add periodic CI works, to ensure PlanSys2 is always up to date.

**Reply by fmrico** [reply]
Please, remember to locally:

```

  git branch -m master rolling
  git fetch origin
  git branch -u origin/rolling rolling
  git remote set-head origin -a
  
```
